jib-core/src/main/java/com/google/cloud/tools/jib/http/FailoverHttpClient.java

```diff
@@ -312,6 +315,19 @@ private Response call(String httpMethod, URL url, Request request, HttpTransport
         httpTransport
             .createRequestFactory()
             .buildRequest(httpMethod, new GenericUrl(url), request.getHttpContent())
+            .setIOExceptionHandler(new HttpBackOffIOExceptionHandler(new ExponentialBackOff()) {
+              @Override
+              public boolean handleIOException(final HttpRequest request, final boolean supportsRetry) throws IOException {
+                final boolean result = super.handleIOException(request, supportsRetry);
+                final String reqString = request.getRequestMethod() + " " + request.getUrl();
```

Comment:
nit per Google-style:
```suggestion
              public boolean handleIOException(HttpRequest request, boolean supportsRetry) throws IOException {
                boolean result = super.handleIOException(request, supportsRetry);
                String requestUrl = request.getRequestMethod() + " " + request.getUrl();
```